tickerview/src/main/java/com/vinay/ticker/lib/TickerView.java

```diff
@@ -37,7 +37,9 @@ public class TickerView extends HorizontalScrollView {
     private TextView lastTicker;
     private List<View> childViews = null;
     private LinearLayout linearLayout;
-
+    private Integer replays;
+    private Integer loop = 1;
+    private boolean replaysCompleted = false;
```

Comment:
Can remove explicit assignment to false since it's already the default value
```suggestion
    private boolean replaysCompleted;
```